Meta/build-root-filesystem.sh

```diff
@@ -12,10 +12,11 @@ window_gid=13
 
 CP="cp"
 
-# cp on macOS does not support the -d option.
+# cp on macOS and FreeBSD do not support the -d option.
 # gcp comes with coreutils, which is already a dependency.
-if [ "$(uname -s)" = "Darwin" ]; then
-	CP=gcp
+OS="$(uname -s)"
+if [ "$OS" = "Darwin" ] || [ "$OS" = "FreeBSD" ] || [ "$OS" = "OpenBSD" ]; then
```

Comment:
Small nitpick, but it'd be cleaner to match for `*BSD` instead, since most (if not all) BSDs don't have this option. [NetBSD's `cp` doesn't](https://man.netbsd.org/NetBSD-9.0-STABLE/i386/cp.1), for example.

Reply:
Agreed. That would be a nice tweak :)

Reply:
Good idea, it's done in my last commit!